packages/components/Callout/macos/RCTCalloutManager.h

```diff
@@ -0,0 +1,5 @@
+#import <React/RCTAssert.h>
```

Comment:
There's no class interface?

Why import all these files if we don't use them?

Reply:
We need most of these. They're used in the Swift code (touch handler, UIManager, etc.). I can remove the import of RCTAssert though